### ISSUE TITLE
Adjust CamelCase names

### ISSUE THREAD

**Issue by bemasc** [MikeBishop/dns-alt-svc#82]
Some of the CamelCase names could be improved.  For example, SvcFieldPriority is now used in AliasForm, so maybe it should just be SvcPriority or SvcIndex.

**Comment by moonshiner** [comment]
These are the CamelCase names I found

SvcDomainName
SvcFieldPriority
SvcFieldValue
SvcParamKey
SvcParamValue
SvcRecordType
AliasForm
ServiceForm
ClientHello

**Comment by marka63** [comment]
Personally I'm happy with SvcFieldPriority.  If it has to change then SvcPriority.

**Comment by enygren** [comment]
Of these I think only SvcFieldPriority might want to change to SvcPriority but I don't strongly care either way.

**Comment by moonshiner** [comment]
SvcPriority makes more sense as it is the priority for the RR itself,

**Comment by tfpauly** [comment]
Agreed that SvcFieldPriority can simply be SvcPriority.

Is there a reason we need "field" in SvcFieldValue? Is SvcValue too value? Could that be SvcParams, to indicate that its the parameter list?

**Comment by moonshiner** [comment]
We should do that, but we will also have to expand the ABNF to ref;ect ot

SVCParams = SVCParam *(SVCParam)

**Comment by bemasc** [comment]
I like "SvcPriority" and "SvcParams".

I wrote up a more radical proposal in #160 that gets rid of all the camel case names.  From that analysis, some comments:

* AliasForm/ServiceForm have generated some confusion because they are in fact a single form as far as parsing is concerned.  I think "mode" is probably more accurate than "form".
* "SvcDomainName" is potentially confusing, given that it appears in both ServiceForm and AliasForm, and can itself contain either ServiceForm or AliasForm records.  Also, "DomainName" seems redundant.
* SvcRecordType is vestigial.  It's now computed implicitly from SvcFieldPriority.  I would rephrase to remove it entirely.

**Comment by enygren** [comment]
Perhaps "TargetName" or "TargetDomain" instead of "SvcDomainName" ?
SvcParams and SvcPriority sound good.
AliasMode/ServiceMode also seem fine if "Form" causes confusion.

**Comment by moonshiner** [comment]
I like "TargetName" - it is a fully qualified domain name and not just a domain name correct ?  
That should be more explicit

**Comment by tfpauly** [comment]
One thing in favor of keeping SvcDomainName is that it is the Alt-Svc name, so having "Svc" is nice.

**Comment by enygren** [comment]
But given the decoupling from Alt-Svc, "TargetName" might actually reduce confusion.

**Comment by tfpauly** [comment]
Fair!

**Comment by bemasc** [comment]
OK, I updated PR #160 to match the names discussed here.  Please review.

**Comment by moonshiner** [comment]
I think you missed defining "SvcParamKey" in the ABNF (or changing display-key to be SvcParamKey). 
not 100%, going to spin a version for myself to walk through it

**Comment by moonshiner** [comment]
Okay, I went back through this and everything looks good Ben.

**Comment by enygren** [comment]
Fixed in #160
